<launch>
  
  <node name="srift_node" pkg="srift_driver" type="srift_node" respawn="false" output="screen" args="192.168.0.110"/>
  

  <node name="srift_force_utils" pkg="srift_utils" type="netft_utils" respawn="false" output="screen" args="/base_link /ft_sensor"  />

</launch>
